NitroxServer/GameLogic/Entities/EntityData.cs

```diff
@@ -11,7 +11,7 @@ namespace NitroxServer.GameLogic.Entities
     [ProtoContract]
     public class EntityData
     {        
-        [ProtoMember(1)]
+        [ProtoIgnore]
```

Comment:
The diff still shows these as changed.  You may want to rebase everything and resubmit the pr.

Reply:
```
discord: btw this is still there...

![](https://cdn.discordapp.com/attachments/635059612000845835/635100777706160149/unknown.png)

You got one file tho
 
PraevalensToday at 9:04 AM
Yes thats because that actually fixed an issue with the entities not being deserialised```

Reply:
> The diff still shows these as changed. You may want to rebase everything and resubmit the pr.

The changes only seem to be reverted in `EscapePodData`, not `EntityData`. Keep in mind that you can (and should!) always (force)push to the same branch _without_ opening a separate PR, losing all this review context.

But if dictionary serialization "works properly" now it is probably best to clean this up for every serializable structure in a separate commit. After all the dictionary itself is still serialized, but with a lock around it. @Sunrunner37 what was the reason for these locks again? A packet should be built and sent on a single thread (if not, add locking where sharing occurs) as this seems pretty overkill to me unless I'm missing something important.

Reply:
So I figured the locking had something to do with it being necessary for saving (serializing) while also continuing to receive packets and save those to the dictionary. But I'm just guessing.

In the mean time I've noticed that the game still doesn't load properly after a server reset so I'll see what i can do about that.

Reply:
@Praevalens any update on this? This issue does really effect working on other issues